.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,22 @@
-Which operating system does the issue occur on?
+For bug reports, the following information can optionally help speed up the process:
 
-If on linux, are you using X11 or Wayland?
+# System
 
+|                   |                                   |
+|-------------------|-----------------------------------|
+| Operating System  | [Linux/BSD/macOS/Windows]         |
+| Alacritty Version | [`alacritty --version` output]    |
+| Display Server    | [X11/Wayland]   (only on Linux)   |
+| Window Manager    | [i3/xfwm/...]   (only on Linux)   |
+| Compositor        | [compton/...]   (only on Linux)   |
+| PTY Backend       | [WinPTY/ConPTY] (only on Windows) |
+
+# Logs
+
+Based on the issue at hand, some logs might be relevant:
+
+| Command                    | Issues                                              |
+|----------------------------|-----------------------------------------------------|
+| STDOUT, STDERR             | Crashes                                             |
+| `alacritty -vv`            | DPI, font size, resize, terminal grid and cell size |
+| `alacritty --print-events` | Problems with keyboard and keybindings              |
```

Comment:
I might add a section below recommending testing `master` if possible.

Reply:
I've addressed the other issues, but thinking about things I'm honestly not too fond of the idea of adding something like this.

Sure, for us it is very nice to make sure bugs still exist on the latest master, however the time it takes for us to close a duplicate issue is so much less than the time it would take someone to clone and build Alacritty, that I do not wish to impose on users like that for something that is likely useless in 90% of the issues.

And if we steer away from lang release cycles like the 0.4.0 release, this should become even less relevant.

Reply:
I agree. In most cases where an issue *is* solved on `master`, @chrisduerr usually slides right in (since he either fixed it himself or merged the PR that did) and asks for that test. I feel there are more times than not where the issue can be closed as a duplicate, or is a new issue that wouldn't benefit from a `master` build anyways.